SpatialGDK/Source/SpatialGDK/Private/Utils/OpUtils.cpp

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Utils/OpUtils.h"
+#include "SpatialConstants.h"
+
+namespace SpatialGDK
+{
+
+	void FindFirstOpOfType(const TArray<Worker_OpList*>& InOpLists, const Worker_OpType InOpType, Worker_Op** OutOp)
```

Comment:
I think our style guide is that there shouldn't be an indent at namespace level.

Reply:
will fix